doc/conf.py

```diff
@@ -20,18 +20,23 @@
 ]
 
 source_suffix = ".rst"
-master_doc = "index"
+root_doc = "index"
 
 autoclass_content = "both"
 autodoc_member_order = "bysource"
 default_role = "py:obj"
 
 html_theme = "sphinx_rtd_theme"
+html_theme_options = {
+    "prev_next_buttons_location": "both",
+}
 htmlhelp_basename = "{}doc".format(project)
 
 language = "en"
 
 napoleon_use_rtype = False
+napoleon_google_docstring = True
+napoleon_numpy_docstring = False
```

Comment:
Out of curiosity, why is this explicitly necessary?

Reply:
`napoleon_numpy_docstring` defaults to true so I was explicitly setting these values within the conf.